pkg/component/runtime/command.go

```diff
@@ -81,25 +93,22 @@ func (c *CommandRuntime) Run(ctx context.Context, comm Communicator) error {
 			switch as {
 			case actionStart:
 				if err := c.start(comm); err != nil {
-					c.forceCompState(client.UnitStateFailed, err.Error())
+					c.forceCompState(client.UnitStateFailed, fmt.Sprintf("Failed: %s", err))
```

Comment:
can we add even more context?

Reply:
All the context should already be coming from the `c.start()`. It adds extra data in those error messages.